### PR TITLE
`create vault` - add create vault and confirm create vault screens

### LINKED ISSUE #1153: `static` - `create vault` - page
#### What would you like to be added:
When you click on create vault, need to fill up these details.
image

Confirm page:
image

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To add create vault screen
- To add confirm create vault screen

#### Which issue(s) does this PR fixes?:

Fixes #1153 

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+function LoanSchemeOptions (props: {loanSchemes: LoanScheme[], selectedLoanScheme?: LoanScheme, onLoanSchemePress: (scheme: LoanScheme) => void}): JSX.Element {
+  return (
+    <View
+      style={tailwind('mb-1')}
+      testID='loan_scheme_options'
+    >
+      {props.loanSchemes.map((scheme, index) => (
+        <ThemedTouchableOpacity
+          key={scheme.id}
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx:149]: should have bg-white for the cards

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  }
+
+  useEffect(() => {
+    client.fee.estimate()
+      .then((f) => setFee(new BigNumber(f)))
+      .catch(logger.error)
+  }, [])
+
+  return (
+    <ThemedScrollView
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx:94]: too much padding on the bottom

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx (hunk 1 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx:149]: updated

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx (hunk 2 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/CreateVaultScreen.tsx:94]: i think you meant the spacing below the button?
Updated, let me know if it's other spacing

**Review by thedoublejay — APPROVED** [review]

**Review by pierregee — APPROVED** [review]

**Pull request merged**